core/commands/files/files.go

```diff
@@ -249,10 +249,10 @@ Examples:
 `,
 	},
 	Arguments: []cmds.Argument{
-		cmds.StringArg("path", false, false, "Path to show listing for. Defaults to '/'."),
+		cmds.StringArg("path", false, false, "Path to show listing for.").Default('/'),
```

Comment:
Also, i just realize this is an argument. Arguments don't have the fancy `Default` decorator yet